admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/service/impl/BulkDataUploadServiceImpl.java

```diff
@@ -169,786 +170,832 @@ public BulkDataGetExtnDto getTrascationDetails(String transcationId) {
+				bulkDataGetExtnDto.setCategory(bulkUploadTranscation.getCategory());
+				bulkDataGetExtnDto.setEntityName(bulkUploadTranscation.getEntityName());
+				bulkDataGetExtnDto.setCategory(bulkUploadTranscation.getCategory());
+				bulkDataGetExtnDto.setUploadedBy(bulkUploadTranscation.getUploadedBy());
+				bulkDataGetExtnDto.setTimeStamp(bulkUploadTranscation.getCreatedDateTime().toString());
+				bulkDataGetExtnDtos2.add(bulkDataGetExtnDto);
+
 			}
-		}catch (Exception e) {
+		} catch (Exception e) {
 			auditUtil.setAuditRequestDto(EventEnum.BULKDATA_TRANSACTION_ALL_ERROR);
-			throw new DataNotFoundException(BulkUploadErrorCode.UNABLE_TO_RETRIEVE_TRANSCATION.getErrorCode(), 
-					BulkUploadErrorCode.UNABLE_TO_RETRIEVE_TRANSCATION.getErrorMessage(),e);
+			throw new DataNotFoundException(BulkUploadErrorCode.UNABLE_TO_RETRIEVE_TRANSCATION.getErrorCode(),
+					BulkUploadErrorCode.UNABLE_TO_RETRIEVE_TRANSCATION.getErrorMessage(), e);
 		}
-		pageDto2 = new PageDto<BulkDataGetExtnDto>(pageData.getNumber(), pageData.getTotalPages(), pageData.getTotalElements(),
-				bulkDataGetExtnDtos2);
+		pageDto2 = new PageDto<BulkDataGetExtnDto>(pageData.getNumber(), pageData.getTotalPages(),
+				pageData.getTotalElements(), bulkDataGetExtnDtos2);
 		return pageDto2;
 	}
 
-
 	private BulkDataResponseDto insertDataToCSVFile(String tableName, String operation, String category,
 			MultipartFile[] files) {
-    		
-			if (tableName.isBlank() || operation.isBlank() || files==null ||files.length==0) {
+
+		if (tableName.isBlank() || operation.isBlank()) {
+			auditUtil.setAuditRequestDto(EventEnum.BULKDATA_INVALID_ARGUMENT);
+			throw new RequestException(BulkUploadErrorCode.INVALID_ARGUMENT.getErrorCode(),
+					BulkUploadErrorCode.INVALID_ARGUMENT.getErrorMessage());
+		}
+		if (files == null || files.length == 0) {
+			auditUtil.setAuditRequestDto(EventEnum.BULKDATA_INVALID_ARGUMENT);
+			throw new RequestException(BulkUploadErrorCode.EMPTY_FILE.getErrorCode(),
+					BulkUploadErrorCode.EMPTY_FILE.getErrorMessage());
+
+		}
+		auditUtil.setAuditRequestDto(EventEnum.getEventEnumWithValue(EventEnum.BULKDATA_UPLOAD,
+				"{category:'" + category + "',tablename:'" + tableName + "',operation:'" + operation + "'}"));
+		BulkDataResponseDto bulkDataResponseDto = new BulkDataResponseDto();
+		mapper.init();
+		Class<?> entity = mapper.getEntity(tableName);
+		String repoBeanName = mapper.getRepo(entity);
+		JobBuilderFactory jobBuilderFactory = new JobBuilderFactory(jobRepository);
+		StepBuilderFactory stepBuilderFactory = new StepBuilderFactory(jobRepository, platformTransactionManager);
+		List<String> failureMessage = new ArrayList<String>();
+		// Map<String,String> failureMessage=new HashMap<String, String>();
+		int[] numArr = { 0 };
+		String[] status = { "PROCESS" };
+		Arrays.asList(files).stream().forEach(file -> {
+			if (file == null || null==file.getContentType() || file.getContentType().length()==0) {

```

Comment:
why is the emtpy_file exception thrown when content_type is not present instead empty_file must be based on the actual file content.

Reply:
Removed this piece of code as was testing file content but never content of file comes empty

Reply:
Removed this piece of code forgot to remove this before pushing  code, while testing  file content never  comes empty was trying to fetch content of file during iteration of list of files but content was never empty